src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -76,6 +86,14 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
     address: this.startingAddress,
     addressErrors: {},
     addressTouched: {},
+    phoneNumber:
+      this.props.order.requestedFulfillment &&
+      (this.props.order.requestedFulfillment.__typename === "CommerceShip" ||
```

Comment:
It was causing type errors for some reason so this is what Sepand and I came up with but I;'m open to alternate suggestions

Reply:
@zephraph @ds300 Any ideas on why this is causing type errors without the check?

Reply:
not sure if I am looking at the right place but is this [`other`](https://github.com/artsy/reaction/blob/073b55f5a30888c2f595010a518dec917387ab67/src/__generated__/ShippingOrderAddressUpdateMutation.graphql.ts#L46) causing it?

Reply:
*tldr* we still need this check and yes @sepans it's to do with that `"%other"`

`"%other"` is a sentinel value that the typescript relay plugin adds to `__typename` union types to force you to handle 'unexpected' cases. Not 100% sure but I think the idea is that in the future you could very possibly add another type to e.g. the union `CommerceRequestedFulfillment`, which wouldn't technically be a breaking schema change, but it could break your code if it's only ever expecting there to be two possible values for `__typename` on `CommerceRequestedFulfillment` and suddenly we add a third.

Reply:
So is it best to leave the check in for now?

Reply:
Yep

Reply:
although now you can change this bit from 

```ts
(this.props.order.requestedFulfillment.__typename === "CommerceShip" ||
        this.props.order.requestedFulfillment.__typename === "CommercePickup")
```

to just

```ts
this.props.order.requestedFulfillment.__typename != "%other"
```

I think.